molecule/default/verify.yml

```diff
@@ -5,5 +5,5 @@
   hosts: all
   tasks:
   - name: Install Kibana
-    package:
+    ansible.builtin.package:
```

Comment:
I could be wrong, but I think older Ansible versions don't like these namespaced module names.

Reply:
I can't see a problem since these are the ansible-lint failures. This is just the FQCN, not the Namespace. The Namespace of this role is "netways".

Do you have more information about your concerns?